packages/ui/styles/globals.css

```diff
@@ -59,8 +59,11 @@
   }
 
   #popup-root {
-    width: 400px;
-    height: 600px;
+    min-width: 400px;
+    min-height: 600px;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
```

Comment:
`min-height` makes the attached popup the correct size, but doesn't restrict the root div from growing larger in the detached popup.